felix/fv/mtu_test.go

```diff
@@ -104,7 +104,7 @@ var _ = infrastructure.DatastoreDescribe("VXLAN topology before adding host IPs
 						Eventually(func() string {
 							out, _ := felix.ExecOutput("cat", "/var/lib/calico/mtu")
 							return out
-						}, "30s", "100ms").Should(ContainSubstring(vxlanEnabledMtu))
+						}, "60s", "100ms").Should(ContainSubstring(vxlanEnabledMtu))
```

Comment:
I wonder if it would make sense to only check the condition every say 500ms or 1s given the huge timeout

Reply:
that's probably a good idea, will do! thanks!